python/datasources/kff_vaccination.py

```diff
@@ -0,0 +1,192 @@
+import pandas as pd
+
+from ingestion.standardized_columns import Race
+import ingestion.standardized_columns as std_col
+
+from datasources.data_source import DataSource
+from ingestion import gcs_to_bq_util, github_util
+
+BASE_KFF_URL_TOTALS_STATE = ('https://raw.githubusercontent.com/KFFData/COVID-19-Data/'
```

Comment:
Do you think this is fragile? Is KFF or GitHub likely to change the way this file is stored? Obviously it's good for now but just thinking we should have it in mind that this might be a point of failure or whatever

Reply:
Yes I do think its fragile but I'm not sure what to do except for wait for it to fail and change it. I could add more error cases.

Reply:
How is it handling an error currently; say for instance KFF renames that file in their GitHub or something; what will a user of our site see ?

Reply:
Nothing, the data pipeline will just fail and we won't be able to update the data until we fix it. The error handling will just be for us to more quickly find the issue.

Reply:
Absolute worst case scenario == The pipeline works but somehow gets the wrong data, and never throw an error to stop it. This is possible I guess but very unlikely.

Reply:
These data pipelines run asynchronously whenever we want them to (when we get new data). They then update the s3 buckets that the frontend talks to. If the pipelines fail then the buckets just dont get updated.

Reply:
Oh right; so error wouldn't really be user facing anyways. Doesn't seem like as big a deal